packages/carbon-graphs/src/js/controls/PairedResult/PairedResult.js

```diff
@@ -78,6 +78,24 @@ const loadInput = (inputJSON) => new PairedResultConfig()
   .clone()
   .getConfig();
 
+/**
+ * Filters invalid data like null and blank.
+ *
+ * @private
+ * @param {object} data - Data points object
+ * @returns {object} filteredData - filtered data object
+ */
+const filterPairedResultData = (data) => {
+  data.map((value) => {
```

Comment:
`Array.map()` returns a new instance of an array so it should be assigned to a variable or associated directly to a return statement as noted in my previous example

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/19/commits/be9685225d23079911dd96b1245cb0ba1b5cea0f#diff-1a3e9e5e1951c8c06a171883626624b21dfb2e6131ec69031c9cf36f5e341f36R89).